libraries/config/include/config/config.hpp

```diff
@@ -71,6 +73,10 @@ struct TestParamsConfig {
 };
 
 struct FullNodeConfig {
+  static const uint32_t kDagExpiryPeriodLimit = 1000;  // Any non finalized dag block with a propose period smaller by
```

Comment:
I have a concern on this design. After introduce PBFT blocks with NULL anchor, we may have lots of empty PBFT blocks when there is no transactions generated. The proposal period only increase when finalizing DAG blocks. When generate empty PBFT blocks, proposal period will not increase but finalized period will keep increase. So the proposal period will be much behind the finalized period

Reply:
Yes, true, I will need to make changes not to count empty pbft blocks.

Reply:
Implemented